repo/fsrepo/migrations/versions.go

```diff
@@ -0,0 +1,89 @@
+package migrations
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"fmt"
+	"os/exec"
+	"path"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/coreos/go-semver/semver"
+)
+
+const distVersions = "versions"
+
+// LatestDistVersion returns the latest version, of the specified distribution,
+// that is available on the distribution site.
+func LatestDistVersion(ctx context.Context, dist string) (string, error) {
+	vs, err := DistVersions(ctx, dist, false)
+	if err != nil {
+		return "", err
+	}
+
+	for i := len(vs) - 1; i >= 0; i-- {
+		ver := vs[i]
+		if !strings.Contains(ver, "-dev") {
+			return ver, nil
+		}
```

Comment:
Not for testing.  This was already present in the code from ipfs-update [here](https://github.com/ipfs/ipfs-update/blob/master/lib/versions.go#L85), so I did not want to remove it in case there was some situation in which it was needed.

Keep it? Leave it?

Reply:
It must have been there for some reason at some time... will leave it for now.